source/_integrations/elmax.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: ELMAX
+description: Instructions on how to integrate Elmax panels within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2022.01
```

Comment:
Sorry my suggestion was wrong! There will not be a 2022.1 release.

```suggestion
ha_release: 2022.2
```

Reply:
Done!